Changelog.md

```diff
@@ -1,3 +1,8 @@
+0.4.0
```

Comment:
I suggest you to do version bumps always in the master branch after a feature is merged. It looks strange if there are that many other commits after creating a changelog with a specific version.